indigo_app/static/javascript/indigo/views/work_commencements.js

```diff
@@ -7,18 +7,24 @@
   /**
    * Handle the work commencements display.
    */
+
+
   Indigo.WorkCommencementsView = Backbone.View.extend({
     el: '#work-commencements-view',
     events: {
       'click .add-commencement': 'addCommencement',
       'click .all-provisions': 'allProvisionsChanged',
       'submit .commencement-form': 'formSubmitted',
+      'change .commencement-form input[name="provisions"]' : 'handleCheckboxesChange',
+      'click .commencement-form .expand-collapse-button' : 'onExpandCollapseClick',
+      'change .commencement-form input[name="select-all"]' : "onSelectAll",
     },
 
     initialize: function() {
       this.model = new Indigo.Work(Indigo.Preloads.work);
-      this.$('.commencement-form').on('show.bs.collapse', _.bind(this.formShow, this));
-      this.$('.commencement-form').on('hide.bs.collapse', _.bind(this.formHide, this));
+
+      // this.$('.commencement-form').on('show.bs.collapse', _.bind(this.formShow, this));
```

Comment:
@longhotsummer can we chat about these functions tomorrow? are they needed?
it causing this issue
https://africanliiteam.slack.com/files/UF69ATL6S/F025J79LDAT/work-commencement.mp4?origin_team=TB3H1VDGD&origin_channel=CK3QE8BU1